lizrd/core/distributed.py

```diff
@@ -54,6 +54,7 @@ def wrap_in_fsdp(
         ),
         cpu_offload=CPUOffload(offload_params=offload_params),
         auto_wrap_policy=wrap_policy,
+        use_orig_params=True,
```

Comment:
From:
https://pytorch.org/docs/stable/fsdp.html#torch.distributed.fsdp.FullyShardedDataParallel
`True is required to use torch.compile()`